Wrapping/Python/LegacyPackaging.cmake

```diff
@@ -2,35 +2,40 @@ if ( SimpleITK_DOC_FILES )
   # create a python list for the import documents to include in
   # packaging
 
-  # This string is targed for setup.py. It will be passed through
-  # the build-time configuration script and as a command line
-  # argument. This sequence is having portability issues with
-  # quote. So Windows needs separate handling from Unix-like
-  # platforms.
-  if( WIN32 )
-    set( _q "'" )
-  else()
-    set( _q "\\'")
-  endif()
-
   # specially handle the first element
   list( GET SimpleITK_DOC_FILES 0 d )
   file(TO_NATIVE_PATH "${d}" d )
-  set( SimpleITK_DOC_FILES_AS_LIST "[r${_q}${d}${_q}")
+  set( SimpleITK_DOC_FILES_AS_LIST "[r'${d}'")
   set( _doc_list "${SimpleITK_DOC_FILES}" )
   list( REMOVE_AT _doc_list 0 )
 
   foreach( d ${_doc_list} )
     file(TO_NATIVE_PATH "${d}" d )
-    set( SimpleITK_DOC_FILES_AS_LIST "${SimpleITK_DOC_FILES_AS_LIST},r${_q}${d}${_q}")
+    set( SimpleITK_DOC_FILES_AS_LIST "${SimpleITK_DOC_FILES_AS_LIST},r'${d}'")
   endforeach()
   set( SimpleITK_DOC_FILES_AS_LIST "${SimpleITK_DOC_FILES_AS_LIST}]")
 
 endif()
 
-include( sitkConfigureFileBuildtime )
-configure_file_buildtime( "${CMAKE_CURRENT_SOURCE_DIR}/Packaging/setup.py.in"
-  "${CMAKE_CURRENT_BINARY_DIR}/Packaging/setup.py" )
+# do initial configuration of CMake variables
+set(SimpleITK_BINARY_MODULE "@SimpleITK_BINARY_MODULE@")
+configure_file(
+  "${CMAKE_CURRENT_SOURCE_DIR}/Packaging/setup.py.in"
+  "${CMAKE_CURRENT_BINARY_DIR}/Packaging/setup.py.in" )
```

Comment:
should this be 'setup.py'  or are you just copying it over to the build directory?